LoRaEngine/deployment.template.json

```diff
@@ -65,6 +65,11 @@
               "image": "${MODULES.LoRaWanPktFwdModule.arm32v7}",
               "createOptions": "  {\r\n  \"HostConfig\": {\r\n    \"NetworkMode\": \"host\", \"Privileged\": true \r\n\r\n  },\r\n  \"NetworkingConfig\": {\r\n    \"EndpointsConfig\": {\r\n      \"host\": {}\r\n    }\r\n  }\r\n}\r\n  \r\n  \r\n  \r\n"
             },
+            "env": {
+              "IOT_SK_SX1301_RESET_PIN": {
+                "value": "5"
```

Comment:
Should we give such a specific name tag or rather a more generic one to be used across device ? (like RESET_PIN)

Reply:
RESET_PIN sound more easy to understand

Reply:
That is why I kept the name. At the end the script is so specific. But I am open to rename it.